website/pages/index.js

```diff
@@ -11,7 +11,7 @@ import PropTypes from 'prop-types';
 
 const Home = ({ navbarContent }) => {
   return (
-    <Layout meta={{ title: 'Welcome' }} navigationContent={navbarContent || {}}>
+    <Layout navigationContent={navbarContent || {}}>
```

Comment:
Maybe you can use defaultProps instead since navbarcontent isn't a required prop.

```js
const Home = ({ navBarContent }) => {
    ...
}
Home.defaultProps = {
  navBarContent: {}
}
Home.propTypes = {
   ...
}
```